golang/Makefile

```diff
@@ -218,6 +218,7 @@ k3d-init-ext: k3d-check
 	k3d cluster create --api-port 172.17.0.1:6443 crane-test && \
 	k3d kubeconfig get crane-test > k3d-auth.yaml && \
 	KUBECONFIG=$(PWD)/k3d-auth.yaml kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/main/bundle.yaml
+	KUBECONFIG=$(PWD)/k3d-auth.yaml kubectl apply -f ./crane-test.yaml
```

Comment:
Revert this please. Add a new make target if you need it. This is used by crane, it is intentional, needed by the stack, if you use prom operator you can spawn service monitors - using its CRDs.

Reply:
Oh okay I thought this was only used by tests.